src/Functions/FunctionsBitmap.cpp

```diff
@@ -9,12 +9,188 @@ namespace DB
+    FunctionDocumentation::Examples examples_bitmapToArray = {{"Usage example", "SELECT bitmapToArray(bitmapBuild([1, 2, 3, 4, 5])) AS res;",
+        R"(
+┌─res─────────┐
+│ [1,2,3,4,5] │
+└─────────────┘
+        )"}
+    };
+    FunctionDocumentation::IntroducedIn introduced_in_bitmapToArray = {20, 1};
+    FunctionDocumentation::Category category_bitmapToArray = FunctionDocumentation::Category::Bitmap;
+    FunctionDocumentation documentation_bitmapToArray = {
+        description_bitmapToArray,
+        syntax_bitmapToArray,
+        arguments_bitmapToArray,
+        returned_value_bitmapToArray,
+        examples_bitmapToArray,
+        introduced_in_bitmapToArray,
+        category_bitmapToArray
+    };
+
+    factory.registerFunction<FunctionBitmapToArray>(documentation_bitmapToArray);
+
+    // Documentation for bitmapSubsetInRange
+    FunctionDocumentation::Description description_bitmapSubsetInRange = "Returns a subset of the bitmap, containing only the elements in the specified range [start, end).";
+    FunctionDocumentation::Syntax syntax_bitmapSubsetInRange = "bitmapSubsetInRange(bitmap, start, end)";
+    FunctionDocumentation::Arguments arguments_bitmapSubsetInRange = {
+        {"bitmap", "Bitmap to extract the subset from. [`AggregateFunction(groupBitmap, T)`](/sql-reference/data-types/aggregatefunction)."},
+        {"start", "Start of the range (inclusive). [`UInt*`](/sql-reference/data-types/int-uint)"},
+        {"end", "End of the range (exclusive). [`UInt*`](/sql-reference/data-types/int-uint)"}
+    };
+    FunctionDocumentation::ReturnedValue returned_value_bitmapSubsetInRange = "Returns a bitmap containing only the elements in the specified range. [`AggregateFunction(groupBitmap, T)`](/sql-reference/data-types/aggregatefunction)";
+    FunctionDocumentation::Examples examples_bitmapSubsetInRange = {{"Usage example", "SELECT bitmapToArray(bitmapSubsetInRange(bitmapBuild([1, 2, 3, 4, 5]), 2, 5)) AS res;",
+        R"(
+┌─res─────┐
+│ [2,3,4] │
+└─────────┘
+    )"}
+    };
+    FunctionDocumentation::IntroducedIn introduced_in_bitmapSubsetInRange = {20, 1};
+    FunctionDocumentation::Category category_bitmapSubsetInRange = FunctionDocumentation::Category::Bitmap;
+    FunctionDocumentation documentation_bitmapSubsetInRange = {
+        description_bitmapSubsetInRange,
+        syntax_bitmapSubsetInRange,
+        arguments_bitmapSubsetInRange,
+        returned_value_bitmapSubsetInRange,
+        examples_bitmapSubsetInRange,
+        introduced_in_bitmapSubsetInRange,
+        category_bitmapSubsetInRange
+    };
+
+    factory.registerFunction<FunctionBitmapSubsetInRange>(documentation_bitmapSubsetInRange);
+
+    // Documentation for bitmapSubsetLimit
+    FunctionDocumentation::Description description_bitmapSubsetLimit = "Returns a subset of a bitmap with smallest bit value `range_start` and at most `cardinality_limit` elements.";
```

Comment:
The term "smallest bit value" is confusing here. Maybe "from position range_start with at most cardinality_limit set bits"?